packages/tasit-account/package.json

```diff
@@ -1,27 +1,39 @@
 {
-  "name": "tasit-account",
-  "version": "0.0.1",
-  "description": "Library for using Ethereum accounts",
-  "keywords": [
-    "ethereum",
-    "dapp",
-    "react",
-    "native"
-  ],
-  "author": "Tasit Labs",
-  "homepage": "https://github.com/tasitlabs/TasitSDK#readme",
-  "license": "MIT",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/tasitlabs/tasitsdk.git"
-  },
-  "scripts": {
-    "test": "echo \"Error: run tests from root\" && exit 1"
-  },
-  "bugs": {
-    "url": "https://github.com/tasitlabs/tasitsdk/issues"
-  },
-  "dependencies": {
-    "ethers": "^4.0.12"
-  }
+	"name": "tasit-account",
```

Comment:
@marcelomorgado For future reference, I would have put this formatting change in a separate PR with no other changes. That way if anything was change din here, it's sure to show up in the diff. GitLab has this nice feature "Ignore whitespace changes" in the diff on their UI that I hope GitHub adds soon.

Reply:
In particular in this PR, it made it hard to notice the async and prepare changes